openwisp_users/api/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.conf.urls import url
+from openwisp_users import settings as app_settings
+
+from . import views
+
+urlpatterns = []
+
+if app_settings.USERS_AUTH_API:
+    urlpatterns += [
+        url(r'^users/token/', views.obtain_auth_token, name='api-token-auth')
+    ]
```

Comment:
for consistency with the other modules, let's use underscores instead of dashes in the `name` argument.

Similar endpoint in django-freeradius: https://github.com/openwisp/django-freeradius/blob/master/django_freeradius/api/urls.py#L15-L20

Let's call it `user_auth_token` as well, we'll separate the two using URLs namespaces (the URLs in the radius module are already namespaced)